controllers/snapshot/snapshot_adapter.go

```diff
@@ -206,14 +254,17 @@ TestScenarioLoop:
 					scenarioLabelAndKey := map[string]string{gitops.SnapshotTestScenarioLabel: integrationTestScenario.Name}
 					binding, err = a.createSnapshotEnvironmentBindingForSnapshot(a.application, &environment, a.snapshot, components, scenarioLabelAndKey)
 					if err != nil {
-						a.logger.Error(err, "Failed to create snapshotEnvironmentbinding for snapshot",
+						errStr := "Failed to create SnapshotEnvironmentBinding for snapshot"
+						a.logger.Error(err, errStr,
 							"snapshot", a.snapshot.Name,
 							"environment.Name", environment.Name,
 							"snapshot.Spec.Components", a.snapshot.Spec.Components)
+						sts.UpdateTestStatusIfChanged(integrationTestScenario.Name, gitops.IntegrationTestStatusEnvironmentProvisionError, errStr)
```

Comment:
@hongweiliu17 I'm not sure if I get this right

`EnvironmentProvisionError` is when we fail to create CR for SEB, or is it an internal status of SEB and should get it from SEB CR directly?

Reply:
When we failed to create environment CR [here](https://github.com/redhat-appstudio/integration-service/blob/main/controllers/snapshot/snapshot_adapter.go#L237)

Reply:
updated